frontend/src/i18n/locales/de/glossary.json

```diff
@@ -181,14 +181,13 @@
   "Access Modes": "Zugriffsmodi",
   "Volume Mode": "Speichervolume-Modus",
   "Storage Class": "Speicher-Klasse",
-  "Volume Claims": "Volume Claims",
+  "Volume Claims": "",
```

Comment:
I think these were mistakenly rewritten as empty (same for all translation languages).